### ISSUE TITLE
Plugin 'VaRest' failed to load because module 'VaRest' could not be found.

### ISSUE THREAD

**Issue by aknarts** [ufna/VaRest#306]
Hi, 

I am packaging my project on Linux and I am getting the following error:
```
   LogModuleManager: Warning: ModuleManager: Unable to load module 'VaRest'  - 0 instances of that module name found.
   LogModuleManager: Warning: ModuleManager: Unable to load module 'VaRest'  - 0 instances of that module name found.
   Message dialog closed, result: Ok, title: Message, text: Plugin 'VaRest' failed to load because module 'VaRest' could not be found.  Please ensure the plugin is properly installed, otherwise consider disabling the plugin for this project.
```

This sounds possibly as a duplicate of https://github.com/ufna/VaRest/issues/294 but I have checked that issue and did not find anything that would match my situation.

Checked the config files and there are several redirects from /Script/VaRestPlugin to /Scrip/VaRest and that is the only mention of the name in the config files that I can see. 

Any ideas where I can start tracking this down?

It fails in the Cook phase and I am on the latest develop branch (7079ff4), with the latest 4.25 from 4.25 branch(the release branch is the same)

**Comment by ufna** [comment]
Hi @aknarts ,

As I understood, you're using plugin from Github as project one. Please try to delete VaRest/Config folder and try to build the project.

**Comment by aknarts** [comment]
Where would that directory be? It is a fresh checkout from github that fails the build so the inis in that directory match what is in the repo here. I did remove it regardless and am trying the build again. 

And yes this is a plugin in <Project>/Plugins directory. In Editor everything works fine, the problem is the packaging. This is a Development build if that makes any difference.

**Comment by ufna** [comment]
Just to be sure, it's a C++ project, not a BP-only?

Could you please provide a small project that reproduces the issue?

**Comment by aknarts** [comment]
It is a C++ project(but VaRest obviously used only in BP), and I can try recreating it but it requries packaging/cooking to manifest so it might take a while.

**Comment by ufna** [comment]
It will be very helpful, cause I can't reproduce it on my side :(

**Comment by aknarts** [comment]
Ok I did not get to the reproduction of the issue(yet) but I have done a work around for it.

So some updates, this is Linux only issue. The workaround is to build the <ProjectName>Editor Development target before the packaging is happening. That way everything works. 

Windows packaging works without issues. I will try to do the reproduction as even if this is not VaRest issue, it is Engine issue and will need that anyway.

**Comment by ufna** [comment]
Can't check compilation from editor on Linux right now, maybe a bit later. And yeah, it seemed to be Engine problem(

**Comment by aknarts** [comment]
I am going to close this as this seems to be fixed in 4.25.1 as far as I know.
